src/app/components/school-menu/school-item.tsx

```diff
@@ -10,28 +10,28 @@ interface SchoolItemProps {
 
 export const SchoolItem = ({ item, index, color }: SchoolItemProps) => {
   return 'description' in item ? (
-    <li key={item.title} className="school-item">
+    <li key={item.title}>
```

Comment:
There is an obvious duplication of code, I see the creation of `li` twice. Can we try to avoid this? Do we need two different components?

Reply:
Refactored SchoolItem